### ISSUE TITLE
run exercise error

### ISSUE THREAD

**Issue by dead-horse** [koajs/kick-off-koa#27]
https://github.com/koajs/koa/issues/407

**Comment by hemanth** [comment]
This is a regression, the last time we solved this was using the `freeport` module.

**Comment by dead-horse** [comment]
@roelof1967  any other exersise failed?

**Comment by roelof1967** [comment]
nope, the exercises before it . did all work well and I could solve it.
